content/AddedListPublicHealthInterventions.introduction.md

```diff
@@ -108,6 +108,7 @@ These can include characteristics such as when infected individuals are contagio
 The development of diagnostic tools is critical to this goal.
 Accurate diagnoses on a large scale is necessary to collect the data needed to develop epidemiological models.
 Other areas of public health that address resource availability, inequity, human behavior, and other components that influence people's exposure to pathogens and ability to manage illness will also be critical to mounting a global response to the pandemic.
+Possible public health intervention types for prevention and control in general include: diagnostics, data gathering, pharmaceutical treatment, pharmaceutical prophylaxis, vaccines, and non-pharmaceutical measures such as social distancing, hand hygiene, cough/sneeze hygiene or practices, contact tracing, isolation, for particularly severe pandemics sometimes the use of different types of facemasks as useful and sometimes measures for reduction of contacts, and personal protective equipment for health workers [@ISBN:978-92-4-156553-0].
```

Comment:
Hi Yael, this is a terrific list! I just had one suggestion- perhaps we could organize these strategies based on their role within the broader pubic health approach? How does this sound? "Public health strategies to manage epidemics include anticipation and early detection of emerging diseases (aided by rapid development of diagnostics), containment (using strategies such as widespread testing, contact tracing, quarantining and isolation), control and mitigation of spread (including social distancing), and elimination and eradication. An effective public health response requires response coordination, disease surveillance and intervention monitoring, risk communication and health education (including the containment of “infodemics” of false information), and health interventions such as vaccines and other pharmaceuticals that can reduce transmission, curb morbidity and mortality, and mitigate the effects on health systems and other sectors of society." ```suggestion
Possible public health intervention types for prevention and control in general include: diagnostics, data gathering, pharmaceutical treatment, pharmaceutical prophylaxis, vaccines, and non-pharmaceutical measures such as social distancing, hand hygiene, cough/sneeze hygiene or practices, contact tracing, isolation, for particularly severe pandemics sometimes the use of different types of facemasks as useful and sometimes measures for reduction of contacts, and personal protective equipment for health workers [@ISBN:978-92-4-156553-0].
```

Reply:
If you'd like to build on this version, Ipsita's suggestion was just that you might want to use the phrase "test, trace and isolate" explicitly :)

Reply:
@rando2 Great! maybe "...containment (using strategies to test, trace and isolate such as widespread testing, contact tracing, quarantining and isolation)" ?

Reply:
thanks @rando2  , @juliettemarie0405 & Ipsita. I'd like to take another look at this again just to double-check but in general looks good to me. I will comment again to let you know if I have any other comments on this last version suggested. For now, noting to ask: For "control and mitigation of spread" did you mean "control and prevention of spread"? I can understand either one. For the parenthetical clause "(including social distancing)" -- that may be the place where we have the option to add a couple more things if we think it's useful like "(including social distancing, personal protective equipment, hand hygiene)"; either way seems ok to me. Also last paragraph seems like an encompassing list from a different perspective, so to make sure it's more encompassing it should include a clause like ", non-pharmaceutical interventions for prevention," maybe right before the health interventions clause. Congrats on the MPH, @juliettemarie0405 !

Reply:
@yemarshall Terrific, thanks for your insight. I used the term "mitigation" because from my understanding measures like social distancing are meant to lessen the impact of an outbreak that has already amplified to become an epidemic/pandemic (whereas "prevention" measures would be occurring before to actually prevent that amplification stage). What do you think? And I agree that PPE and hand hygiene should be added to the list- perhaps we could add it to the parentheses after health education, since the education about PPE and hand hygiene is the actual PH intervention? So it could read "...and health education (including education about personal protective equipment and hand hygiene and containment of “infodemics” of false information)" And yes, I agree about adding non-pharmaceutical interventions to the second list! (And thank you for the congratulations!)

Reply:
@juliettemarie0405 (1) Thanks for explaining your understanding of some of the terms; sometimes it's easier to understand terms better from context and I don't have a strong opinion on these. We hear the particular word "prevention" so much so that's really the main reason I noticed the lack of the term "prevention" and that's sort of iffy reasoning, I realize. I was thinking both prevention on individual-scale like prevent transmission from infectious or individual measures to prevent getting infection & you also pointed out in your comment population-scale-considerations to prevent spread (whether local epidemics or pandemics); since these paragraphs include an overview of infectious disease public health, i thought maybe it made sense to include the actual word "prevention" somehow though I know it is already within the paragraphs just using other terms. But again my reasoning is sort of iffy, so this is not a big deal, and whatever you think is good works for me. (2&3) Great, so within your framework/paragraphs it makes sense to me too where you said we could add PPE & hand hygiene and where you/we said we could add mention of non-pharmaceutical interventions.  I think you'd be better than me at adding these few additional adjustments within the paragraphs/organization you put together.

Reply:
@rando2, @juliettemarie0405 & I agree on just a couple more things to add in to what @juliettemarie0405 put above as a suggested change to my pull request; these are straightforward so any of us can insert them. Here they are exactly:
- In the 2nd paragraph added above replace extend the parentheses after health education so that reads "...and health education (including education about personal protective equipment and hand hygiene and containment of “infodemics” of false information),..."
- Also in the 2nd paragraph, right before "health interventions" add in "non-pharmaceutical interventions" so that it reads: "..., non-pharmaceutical measures to prevent and reduce transmission, and health interventions such as..." {the latter addition is very similar wording to the reference & also in general what's written in this PR & in the suggestion above is similar to the ref} OR if you really prefer more simply then "..., non-pharmaceutical interventions and health interventions such as..." but this latter way of adding may get confusing depending on how you interpret "health intervention" and also b/c both non-pharmaceutical & the health intervention examples listed can potentially reduce transmission (directly &/or indirectly) while only the health intervention examples listed can directly reduce morbidity & mortality.
- you can ignore my comments above related to "prevention"/"prevent" b/c it's implied (& b/c also I hadn't remembered the similar categorization used in the reference until I looked at it again :-P ).